extlinks/organisations/templates/organisations/organisation_detail.html

```diff
@@ -55,6 +55,14 @@ <h1>{{ object }}</h1>
                                     {% endif %}
                                 </div>
                             {% endif %}
+                        <div class="form-row">
```

Comment:
My final (I promise!) request is that the 'Exclude bots' option should be on the same row as the user list checkbox. I think that should be solvable by just removing this row div, here and on the program page.